### ISSUE TITLE
No mod tab on penumbra

### ISSUE THREAD

**Issue by ffxivgamergirl** [xivdev/Penumbra#335]
Hello, I am trying to figure out why i dont have a mod tab on my penumbra?

**Comment by Theo-Asterio** [comment]
Please read the reniguide. [https://reniguide.info/#installpenumbra](url) You need to setup Penumbra with a root directory.

**Comment by ffxivgamergirl** [comment]
its all set up.

**Comment by Theo-Asterio** [comment]
Nope, you'd have a mod tab if you did. Pick a root directory and read the entire guide instead of skipping.

**Comment by ffxivgamergirl** [comment]
i have a root directory im not sure why there is no tab for my mods in penumbra

**Comment by Theo-Asterio** [comment]
You did not, set the root directory. Show your penumbra window and use the Discord for support.

**Comment by SerenitySublime** [comment]
https://discord.gg/kVva7DHV4r

**Comment by NotNite** [comment]
Moving all support to a Discord server renders GitHub issues pointless; I'll pass. You didn't set up your mod directory correctly and are incorrect about the fact that it's "all set up". This issue should be closed as invalid, because it's user error.
